src/lib/images/proxy.ts

```diff
@@ -152,7 +150,12 @@ export const initializeProxy = async function ({
       console.log(`fullImageUrl: ${fullImageUrl}`)
       req.url = `/${modifiers}/${fullImageUrl}`
     } else {
-      // If the image is remote, we can just pass the URL as is
+      // If the image is remote, we first check if it's allowed by any of patterns
+      if (!remoteImages.some((remoteImage) => remoteImage.test(sourceImagePath))) {
+        res.status(403).send('Not allowed Remote Image URL.')
```

Comment:
Not sure what platform does for not allowed - so probably this should match platform. For now status matches one from `ipx` and message is a bit custom to match our config field name

Reply:
adjusted it to 400 status without body which seems to be behavior on the platform